tests/congruence_tests/test_search.py

```diff
@@ -144,27 +144,46 @@ def filter_search_text_single(
             limit=10,
         )
 
+    def load_query_vectors(self, query):
+        self.query_text = np.array(query.vector['text']).tolist()
+        self.query_image = np.array(query.vector['image']).tolist()
+        self.query_code = np.array(query.vector['code']).tolist()
+
 
 def test_simple_search():
     fixture_points = generate_fixtures()
-
+    query, fixture_points = fixture_points[-1], fixture_points[:-1]
     searcher = TestSimpleSearcher()
+    searcher.load_query_vectors(query)
 
     local_client = init_local()
-    init_client(local_client, fixture_points)
+    init_client(local_client, fixture_points, init_debug_collection=True)
 
     remote_client = init_remote()
-    init_client(remote_client, fixture_points)
-
-    compare_client_results(local_client, remote_client, searcher.simple_search_text)
-    compare_client_results(local_client, remote_client, searcher.simple_search_image)
-    compare_client_results(local_client, remote_client, searcher.simple_search_code)
-    compare_client_results(local_client, remote_client, searcher.simple_search_text_offset)
-    compare_client_results(local_client, remote_client, searcher.simple_search_text_with_vector)
-    compare_client_results(local_client, remote_client, searcher.search_score_threshold)
-    compare_client_results(local_client, remote_client, searcher.simple_search_text_select_payload)
-    compare_client_results(local_client, remote_client, searcher.simple_search_image_select_vector)
-    compare_client_results(local_client, remote_client, searcher.search_payload_exclude)
+    init_client(remote_client, fixture_points, init_debug_collection=True)
+
+    remote_client.upload_points(COLLECTION_NAME + '_debug', [query], wait=True)
+
+    try:
+        compare_client_results(local_client, remote_client, searcher.simple_search_text)
+        compare_client_results(local_client, remote_client, searcher.simple_search_text_offset)
+        compare_client_results(local_client, remote_client, searcher.simple_search_text_with_vector)
+        compare_client_results(local_client, remote_client, searcher.search_score_threshold)
+        compare_client_results(local_client, remote_client, searcher.simple_search_text_select_payload)
+        compare_client_results(local_client, remote_client, searcher.search_payload_exclude)
+    except AssertionError as e:
+        raise ValueError("Vector:", searcher.query_text) from e
+
+    try:
+        compare_client_results(local_client, remote_client, searcher.simple_search_image)
+        compare_client_results(local_client, remote_client, searcher.simple_search_image_select_vector)
+    except AssertionError as e:
+        raise ValueError("Vector:", searcher.query_image) from e
+
+    try:
+        compare_client_results(local_client, remote_client, searcher.simple_search_code)
+    except AssertionError as e:
+        raise ValueError("Vector:", searcher.query_code) from e
```

Comment:
this seems to be not very useful, cause tests usually being run and fail in CI, and there we don't have an access to the data and checking queries does not make a lot of sense